client-side/src/components/report.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import GenericButton from '../stories/Button/GenericButton';
+import DateInputs from "../stories/DateTime/DateInputs";
+const DateTimePicker = () => {
+    const [startDate, setStartDate] = useState('');
+    const [endDate, setEndDate] = useState('');
+    const storeArr = () => {
+            const dataArr = [startDate, endDate];
+             //fillData(dataArr);
+       
+    }
+       
+
+    const handleStartDateChange = (start) => {
+        console.log('Start Date Selected:', start);
+        setStartDate(start);
+    };
+    const handleEndDateChange = (end) => {
+        console.log('End Date Selected:', end);
+        setEndDate(end);
```

Comment:
you should use one function for handleStartDateChange and handleStartDateChange, something like:
const handleDateChange = (inputName, date) => {
   if(inputName === 'startDate'){
      setStartDate(date);
   } else {
      setEndDate(date);
   }
}